pyproject.toml

```
@@ -87,6 +87,7 @@ shapely = { version = ">=2,<3", optional = true }
 # issues with versions <3.0.2
 snowflake-connector-python = { version = ">=3.0.2,<4,!=3.3.0b1", optional = true }
 trino = { version = ">=0.321,<1", optional = true }
+e6data-python-connector = { version = "2.2.0" }
```

```suggestion
e6data-python-connector = { version = ">=2.2.0,<3", optional = true }
```

Unless there's a reason to pin to a single version, this should include a version range, and `optional = true` is necessary to ensure this is shipped as an extra, and not a required dependency to use any other Ibis backend(s).